src/constants/feature-lists-data.ts

```diff
@@ -1,35 +1,209 @@
 //this is the data for the feature list component and developer can add the features in this file
 const featureListData = {
   android: {
-    categories: []
+    categories: [
+      {
+        items: [
+          {
+            content:
+              'Add auth to apps using the Amplify CLI and Studio. Support for login, MFA, social providers, and CAPTCHA protection. Integrate auth with intuitive client library APIs.',
+            linkText: 'Simple Configuration',
+            link: '/build-a-backend/auth/set-up-auth/'
+          },
+          {
+            content:
+              'Quickly add polished auth and account UIs with <Authenticator> and <AccountSettings> UI components. Integrate seamlessly with minimal code.',
+            linkText: 'Pre-built UI components',
+            link: 'https://ui.docs.amplify.aws/android/connected-components/authenticator'
+          },
+          {
+            content:
+              'Manage users and groups directly in Amplify Studio without writing code. Maintain full control of your user base.',
+            linkText: 'User management',
+            link: '/tools/console/auth/user-management/'
+          }
+        ],
+        heading: 'Auth'
+      },
+
+      {
+        items: [
+          {
+            content:
+              "Amplify provides advanced, intuitive data modeling capabilities. Define your API using our the schema.graphql file in the CLI or Amplify Studio's visual editor. Model relationships between types, customize fields, and configure validation rules. Amplify handles turning the schema into fully implemented backend and frontend code.",
+            linkText: 'Powerful data modeling',
+            link: 'build-a-backend/graphqlapi/data-modeling/'
+          },
+          {
+            content:
+              'Fetch and mutate data through generated GraphQL queries and mutations from your frontend. Subscriptions allow real-time data sync. Integrate backend data sources like DynamoDB easily.',
+            linkText: 'Seamless real-time data access',
+            link: 'build-a-backend/graphqlapi/subscribe-data/'
+          },
+          {
+            content:
+              'Implement fine-grained authorization rules and policies to securely access data. Control auth at the API level or on individual fields. Manage access with AWS IAM policies or your own auth provider.',
+            linkText: 'Granular Authorization',
+            link: 'build-a-backend/graphqlapi/customize-authorization-rules/'
+          }
+        ],
+        heading: 'Data'
```

Comment:
We need to remove the label "Data" for current gen content and only use it for Gen 2. Use "GraphQL API" instead.

Reply:
Got it, thank you

Reply:
Updated @renebrandel